src/components/search/refinement-list.tsx

```diff
@@ -30,6 +30,7 @@ const TagItem: FunctionComponent<
         href={createURL(item.value)}
         onClick={(event) => {
           event.preventDefault()
+          item.isRefined = !item.isRefined
```

Comment:
This was done because algolia's `refine` function was correctly refining the list, but it wasn't flipping the `isRefined` boolean on these items.